packages/rs-drive-abci/src/execution/validation/state_transition/state_transitions/identity_credit_withdrawal/balance/v0/mod.rs

```diff
@@ -30,7 +30,7 @@ impl IdentityCreditTransferTransitionBalanceValidationV0 for IdentityCreditWithd
         &self,
         identity: Option<&mut PartialIdentity>,
         platform: &PlatformStateRef,
-        block_info: &BlockInfo,
+        _unused_block_info: &BlockInfo,
```

Comment:
Are we using this param in any function? Maybe just remove it?

Reply:
imo; best to keep as _unused* atm; may want to revisit in future PRs. This keeps diff smaller